ios/MLRN/MLRNVectorSource.m

```diff
@@ -5,7 +5,11 @@ @implementation MLRNVectorSource
 - (nullable MLNSource*)makeSource
 {
     if (self.url != nil) {
+      if([self.url hasPrefix:@"pmtiles://"]) {
+        return [[MLNVectorTileSource alloc] initWithIdentifier:self.id tileURLTemplates:@[self.url] options:nil];
```

Comment:
No options? Is that because self.url contains all the params?

Reply:
Valid request, as it allows to pass the attribution, tested and works. Min/max zoom should be delivered through PMTiles, but it might also be possible to overwrite this way.